src/redux/actions/projects/deleteProject.js

```diff
@@ -71,6 +71,10 @@ const deleteProject = (
       },
     });
 
+    dispatch({
+      type: BACKEND_STATUS_DELETED,
```

Comment:
You're only using `BACKEND_STATUS_DELETED` to react to an experiment getting deleted, right? I think you could reuse the `EXPERIMENTS_DELETED` action, look into this: https://redux.js.org/style-guide/style-guide#allow-many-reducers-to-respond-to-the-same-action.

Basically you can just add in backend status a reducer for EXPERIMENTS_DELETED, that way you can get rid of `BACKEND_STATUS_DELETED`.

There is a good example of this in `EXPERIMENT_SETTINGS_PIPELINE_START`, you can see it is handled across a lot of small reducers. 

Ideally we should move away from dispatching many actions sequentially for one action creator. I think the problem is that we were creating and using redux actions to modify the store while we should instead have think of the store as reacting as a response to actions.

Reply:
Yeah I thought of that, but in the future we will have many experiments associated to a project right? So in this case we might need to use this backend_status_deleted in other places (when you delete a particullar experiment, not the entire project)

Reply:
What do you think?